server/src/main/java/org/apache/druid/server/log/DefaultRequestLogEvent.java

```diff
@@ -56,7 +57,21 @@
   @Override
   public Map<String, Object> toMap()
   {
-    return ImmutableMap.of();
+    final Map<String, Object> map = new HashMap<>();
+    map.put("feed", getFeed());
+    map.put("timestamp", getCreatedTime());
+    map.put("service", getService());
+    map.put("host", getHost());
+    if (getQuery() != null) {
+      map.put("query", getQuery());
+    }
+    if (getSql() != null) {
+      map.put("sql", getSql());
+    }
+    map.put("sqlQueryContext", getSqlQueryContext());
```

Comment:
Here we need to distinguish between forNative and forSql.
`forNative` : sql and sqlQueryContext are null,
`forSql` : query is null.
So that maybe we need to do 
```
    if (getSqlQueryContext() != null) {
      map.put("sqlQueryContext", getSqlQueryContext());
    }
```

Reply:
done, thanks